up-linux-streamer/Cargo.toml

```diff
@@ -9,17 +9,20 @@ keywords.workspace = true
 license.workspace = true
 
 [dependencies]
-tokio = { version = "1.35.1", default-features = false }
-up-rust = { git = "https://github.com/eclipse-uprotocol/up-rust", rev = "3a50104421a801d52e1d9c68979db54c013ce43d" }
+async-trait = { workspace = true }
+clap = { version = "4.5.9", features = ["derive"] }
+env_logger = { version = "0.10.2" }
+log = { workspace = true }
+json5 = { workspace = true }
+protobuf = { workspace = true }
+serde = { workspace = true }
+tokio = { workspace = true }
```

Comment:
tokio has no features enabled by default, so I guess you will need to activate the needed features here ...

Reply:
They're enabled at the workspace level in that [Cargo.toml](https://github.com/eclipse-uprotocol/up-streamer-rust/pull/16/files#diff-2e9d962a08321605940b5a657135052fbcef87b5e360662bb527c96d9a615542R41), unless I'm missing something :thinking:

Reply:
Calling this one resolved unless you think otherwise. Feel free to reopen if so.